### PR TITLE
cli: amend cli commands for consistency

### PR DESCRIPTION
**Pull request by diegodelemos**

### CODE REVIEW

File: reana_client/cli/analyses.py [hunk 1]
```diff
@@ -38,7 +38,9 @@ def analyses(ctx):
     logging.debug('analysis')
 
 
-@click.command()
+@click.command(
+    'validate',
+    help='Validate reana specification file.')
```
Comment (tiborsimko) [inline reana_client/cli/analyses.py]: `s/reana/REANA/g`

File: reana_client/cli/analyses.py [hunk 2]
```diff
@@ -48,7 +50,7 @@ def analyses(ctx):
     help='REANA specifications file describing the workflow and '
          'context which REANA should execute.')
 @click.pass_context
-def analysis_validate(ctx, file):
+def analyses_validate(ctx, file):
```
Comment (tiborsimko) [inline reana_client/cli/analyses.py]: The plural will read a bit strange here... What about simple `validate`?  We don't use "analyses" much, and the "analysis environment" might be defined by `env`... Let's muse IRL.

File: reana_client/schemas/reana_analysis_schema.json [hunk 3]
```diff
@@ -116,23 +116,29 @@
...
-        "type": {
-          "id": "/properties/workflow/properties/type",
-          "type": "string",
-          "title": "Workflow engine.",
-          "description": "Name which represents a supported workflow engine to be used to reproduce the analysis."
+    "workflows": {
+      "id": "/properties/workflows",
+      "type": "array",
+      "title": "Analysis workflows.",
+      "description": "Set of workflow which represent the steps that need to be run to reproduce an analysis.",
```
Comment (hjhsalo) [inline reana_client/schemas/reana_analysis_schema.json]: "Set of workflow**s**"?

File: reana_client/schemas/reana_analysis_schema.json [hunk 4]
```diff
@@ -116,23 +116,29 @@
...
+            "id": "/properties/workflows/items/properties/file",
+            "type": "string",
+            "title": "Workflow file name.",
+            "default": "workflow.yml"
+          },
+          "type": {
+            "id": "/properties/workflows/items/properties/type",
+            "type": "string",
+            "title": "Workflow engine.",
+            "description": "Name which represents a supported workflow engine to be used to reproduce the analysis."
```
Comment (hjhsalo) [inline reana_client/schemas/reana_analysis_schema.json]: "... reproduce the analysis / run the workflow"?

File: reana_client/schemas/reana_analysis_schema.json [hunk 5]
```diff
@@ -116,23 +116,29 @@
...
+      "items": {
+        "id": "/properties/workflows/items",
+        "type": "object",
+        "title": "Analysis workflow.",
+        "description": "Workflow which represent the steps that need to be run to reproduce an analysis.",
+        "properties": {
+          "file": {
+            "id": "/properties/workflows/items/properties/file",
+            "type": "string",
+            "title": "Workflow file name.",
```
Comment (hjhsalo) [inline reana_client/schemas/reana_analysis_schema.json]: It is fine as is, but for some reason it got me thinking should it be changed to something like:
"Name of workflow definition file."?
"Workflow definition file name"?
"Workflow definition/specification filename"?

Just did a quick search on reanahub-organization and it seems that 'file name' is used more than 'filename'.

Comment (diegodelemos) [reply reana_client/schemas/reana_analysis_schema.json]: Workflow specification filename. Thanks! 👍

**Comment by diegodelemos** [comment]
Closing as discussed IRL with @tiborsimko and @hjhsalo. We will go for:

`reana.yaml` 1 <--> 1 `workflow run`

**Pull request closed without merging**